features/config/service/api/src/main/java/org/opennms/features/config/service/api/ConfigurationManagerService.java

```diff
@@ -60,6 +63,8 @@
     <ENTITY> void registerSchema(String configName, int majorVersion, int minorVersion,
                                  int patchVersion, Class<ENTITY> entityClass) throws IOException, JAXBException;
 
+    <ENTITY> void registerSchema(String configName, Version version, Class<ENTITY> entityClass) throws IOException, JAXBException;
```

Comment:
I changed the interface slightly to group the version information into one object. @freddy61025 do you think this makes sense?